iphone/Classes/TiBluetoothPeripheralManagerProxy.m

```diff
@@ -159,7 +159,7 @@ - (void)updateValueForCharacteristicOnSubscribedCentrals:(id)args
         [result addObject:[(TiBluetoothCentralProxy *)subscribedCentral central]];
     }
     
-    [peripheralManager updateValue:[(TiBlob *)value data]
+    return [peripheralManager updateValue:[(TiBlob *)value data]
```

Comment:
Yes it works for me, but I agree with you. Can you help me on this? not sure how to implement that